modules/project_cleanup/function_source/main.go

```diff
@@ -372,6 +460,41 @@ func invoke(ctx context.Context) {
 		}
 	}
 
+	///r
+	removeFeedsByName := func(ctx context.Context, client *asset.Client, organization, feedName string) (string, error) {
+		logger.Printf("Try to remove feeds with name [%s] from organization [%s]", feedName, organization)
+
+		randomIDPattern := `^organizations/` + organization + `/feeds/fd-cai-monitoring-[0-9a-f]{2}$`
+		regex, err := regexp.Compile(randomIDPattern)
+		if err != nil {
+			return "", fmt.Errorf("Failed to compile the regular expression: %v", err)
+		}
+
+		req := &assetpb.ListFeedsRequest{
+			Parent: fmt.Sprintf("organizations/%s", organization),
+		}
+
+		resp, err := client.ListFeeds(ctx, req)
+		if err != nil {
+			return "", fmt.Errorf("Failed to list feeds from the organization: %v", err)
+		}
+
+		for _, feed := range resp.Feeds {
+			if regex.MatchString(feed.Name) {
+				delReq := &assetpb.DeleteFeedRequest{
+					Name: feed.Name,
+				}
+
+				if err := client.DeleteFeed(ctx, delReq); err != nil {
+					return feedName, fmt.Errorf("Failed to remove the feed %s: %v", feed.Name, err)
+				}
+				logger.Printf("Feed [%s] successfully removed.", feed.Name)
+			}
+		}
```

Comment:
see https://github.com/renato-rudnicki/terraform-google-scheduled-function/blob/37c61438b881f5fdca03b2c03f5992ea5c545853/modules/project_cleanup/function_source/main.go#L452C3-L460C4

it should use  `checkIfCaiFeedsShortNameIncluded`
and  `caiFeedAgeFilter(feed)`